src/transformers/models/mllama/modeling_mllama.py

```diff
@@ -1902,14 +1902,12 @@ def forward(
         full_text_row_masked_out_mask: Optional[Tuple[torch.Tensor, torch.Tensor]] = None,
         past_key_values: Optional[Union[Cache, List[torch.FloatTensor]]] = None,
         inputs_embeds: Optional[torch.FloatTensor] = None,
-        labels: Optional[torch.LongTensor] = None,
```

Comment:
in order for common tests to pass, I may have to keep this and leave it unused. the common tests pass labels to forward for that CausalLM class mapping types , so it results in invalid argument otherwise. Let me know if any better ideas for now that are aligned with any refactors planned